tests/Tokenizer/TokensTest.php

```diff
@@ -888,6 +888,27 @@ public function provideEnsureWhitespaceAtIndexCases()
     {
         return [
             [
+                '<?php echo 1;',
```

Comment:
thanks :)
>$this->assertTrue($tokens->isChanged()

this was incorrect of me, as not all tests make a change, sorry about that :(